.dockerignore

```diff
@@ -1,5 +1,9 @@
 .git
-.gradle
 .gradletasknamecache
 *.jar
 **/build/
+
+# Should exclude the whole .gradle and un-exclude .gradle/tomcat, but the Gradle plugin doesn't support it
+.gradle/4*
+.gradle/buildOutputCleanup
+.gradle/hsqldb
```

Comment:
there is also a `overlays/uPortal/.gradle/nodejs` folder that will exist when there is a custom skin.

Reply:
I'll test the `.gradle/[0-9A-Za-su-z]*` approach.